javascript/javascript1/week2/weatherWear.js

```diff
@@ -0,0 +1,18 @@
+
+function whatToWear(temprature){
+    var str='';
```

Comment:
never use var to define as it's allow to use variable globally through out the code.